### ISSUE TITLE
[exporter/awscloudwatchlogs] invalid_num_consumers unit test failure

### ISSUE THREAD

**Issue by crobert-1** [open-telemetry/opentelemetry-collector-contrib#27624]
### Component(s)

exporter/awscloudwatchlogs

## Description
CI/CD Test is failing

```
--- FAIL: TestLoadConfig (0.01s)
    --- FAIL: TestLoadConfig/awscloudwatchlogs/invalid_num_consumers (0.00s)
        config_test.go:102: 
            	Error Trace:	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter/config_test.go:102
            	Error:      	Error "number of queue consumers must be positive; number of queue consumers must be positive" does not contain "'sending_queue.num_consumers' must be 1 or greater"
            	Test:       	TestLoadConfig/awscloudwatchlogs/invalid_num_consumers
FAIL
	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter	coverage: 88.5% of statements
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter	0.076s
```
[Failure link](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/6484538391/job/17608643006?pr=27608)

### Collector version

v0.87.0

## Environment
OS: (e.g., "Ubuntu 20.04")
Compiler(if manually compiled): (e.g., "go 14.2")

**Comment by songy23** [comment]
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/26692 looks related

**Comment by bryan-aguilar** [comment]
Working on this now, PR should follow shortly.

**Comment by bryan-aguilar** [comment]
fixed in #27625
